README.md

```diff
@@ -64,7 +64,7 @@
   <a href="https://gatsby.dev/discord">Discord</a>
 </h2>
 
-Gatsby is a modern web framework for blazing fast websites.
+Gatsby helps professional developers efficiently create maintainable, highly-performant, content-rich websites
```

Comment:
Do we still want to keep the period at the end?

Reply:
Yes - good call, let's keep that too.